docker/docker-compose.pgsql.yml

```diff
@@ -0,0 +1,27 @@
+services:
+  mapbender:
+    build:
+      context: ../
+    ports:
+      - 80:8080
+    environment:
+      MAPBENDER_DATABASE_URL: "pgsql://pguser:pgpass@db:5432/postgres"
+    depends_on:
+      db:
+        condition: service_healthy
+  db:
+    image: postgis/postgis:14-3.4
+    restart: always
+    ports:
+      - 55432:5432
+    #volumes:
+    #  - ./volumes/db/data:/var/lib/postgresql/data:rw
+    environment:
+      POSTGRES_USER: pguser
```

Comment:
shouldn't these values be configurable?

Reply:
I think there is a lot of value in the simplicity of this self contained configuration and we should keep it like this for the current release.
For future releases we should sit together and derive a dev setup that can be used for development which will then be configurable.

Reply:
alright, sounds like a reasonable approach